warrior/Actions/CommonActions/common_actions.py

```diff
@@ -433,5 +435,9 @@ def get_time_delta(self, start_time, end_time=None, time_diff="time_diff"):
         time_delta = datetime_utils.get_time_delta(start_time=start_time, end_time=end_time)
         print_info("delta between given timestamps : {0} seconds".format(time_delta))
         output_dict = {time_diff: time_delta}
+        if max_time_diff:
+            if time_delta > int(max_time_diff):
+                print_info("The time difference is greater than max time difference so failing the step !")
+                return False
```

Comment:
```output_dict``` is not being returned here. Subsequent failures may occur because of this.

Reply:
Hi @sanikakulkarni 

Made necessary changes and committed.